pkg/patch/patch.go

```diff
@@ -94,6 +94,11 @@ func patchWithContext(ctx context.Context, image, reportFile, patchedTag, workin
 			patchedTag = fmt.Sprintf("%s-%s", tag, defaultPatchedTagSuffix)
 		}
 	}
+	_, err = reference.WithTag(imageName, patchedTag)
+	if err != nil {
+		log.Errorf("invalid patched tag %s", patchedTag)
+		return err
```

Comment:
can we avoid the double log and wrap and return the error

something like
```suggestion
		return fmt.Errorf("%w with patched tag %s", err, patchedTag)
```

Reply:
![image](https://github.com/project-copacetic/copacetic/assets/13926417/7d7f3efa-1b21-4ab1-a8d2-52095b32a2a2)